### PR TITLE
[WIP/ENH] Allows list of references and/or targets in define_target_events

### PR DESCRIPTION
**Pull request by claire-braboszcz**
Modifying the function define_target_events so it can accept lists as arguments (for example when you have multiple names coding for  events/responses and want to select events by response time).
For now it works only if new_id is defined.
It also passes the current test (with single int as value for reference and target) but a new one with lists as argument still has to be defined.
I'm fairly new to python so any advice/comment is welcome :-)

### CODE REVIEW

File: mne/event.py [hunk 1]
```diff
@@ -72,24 +72,24 @@ def pick_events(events, include=None, exclude=None, step=False):
...
     events : ndarray
         Array as returned by mne.find_events.
-    reference_id : int
-        The reference event. The event defining the epoch of interest.
-    target_id : int
-        The target event. The event co-occurring in within a certain time
-        window around the reference event.
+    reference_ids : list[int] or int
+        The reference events. The events defining the epoch of interest.
+    target_ids : list[int] or int
```
Comment (kingjr) [inline mne/event.py]: 'list of int', 

same below

File: mne/event.py [hunk 2]
```diff
@@ -109,26 +109,36 @@ def define_target_events(events, reference_id, target_id, sfreq, tmin, tmax,
...
     new_events = []
     lag = []
+    index = 0
     for event in events.copy().astype(int):
-        if event[2] == reference_id:
+        if event[2] in reference_ids:
             lower = event[0] + imin
             upper = event[0] + imax
+            matching_events = reduce(np.logical_or,
+                                    [events[:,2] == target for target in target_ids])
```
Comment (kingjr) [inline mne/event.py]: np.any([events[:, 2] == target for target in targets_ids])

**Comment by kingjr** [comment]
Hi @claire-braboszcz , and welcome.

Can you clarify what this is useful for by adding a snippets here?

There's some flake8 issues with your code, checkout http://mne-tools.github.io/stable/contributing.html?highlight=pep8#id3

Also, you'll need to add a test.

**Comment by claire-braboszcz** [comment]
```python
def hello():

```

**Comment by claire-braboszcz** [comment]
Thank you for the quick code review! I've implemented the suggested changes. Test and code snippet to follow.

File: mne/event.py [hunk 3]
```diff
@@ -72,24 +72,24 @@ def pick_events(events, include=None, exclude=None, step=False):
     return events
 
 
-def define_target_events(events, reference_id, target_id, sfreq, tmin, tmax,
+def define_target_events(events, reference_ids, target_ids, sfreq, tmin, tmax,
```
Comment (jasmainak) [inline mne/event.py]: you cannot do this without deprecation

Comment (kingjr) [inline mne/event.py]: You actually don't need to rename your vars do you?

**Comment by kingjr** [comment]
@claire-braboszcz when you modify a branch, you can verify whether it will pass the tests using `nosetests`.

Here Travis isn't happy: https://travis-ci.org/mne-tools/mne-python/jobs/135729784, so you'll need to fix that.

Also, if you enhance the API, you need to document it in `mne-python/doc/whats_new.rst/`

File: mne/event.py (hunk 3 above)
Comment (agramfort) [inline mne/event.py]: yes don't do this. revert

**Comment by jona-sassenhagen** [comment]
I like this feature.

File: mne/event.py [hunk 4]
```diff
@@ -109,25 +109,32 @@ def define_target_events(events, reference_id, target_id, sfreq, tmin, tmax,
...
 
     new_events = []
     lag = []
+
     for event in events.copy().astype(int):
-        if event[2] == reference_id:
+        if event[2] in reference_ids:
             lower = event[0] + imin
             upper = event[0] + imax
+            matching_events = np.any([events[:, 2] == target for target in targets_ids])
```
Comment (jona-sassenhagen) [inline mne/event.py]: Wouldn't `events[:, 2] in target_ids` be sufficient?

Comment (agramfort) [inline mne/event.py]: no but it's maybe more readable with set. Something like:

len(set(events[:, 2]).intersection(targets_ids)) > 0

File: mne/event.py (hunk 3 above)
Comment (dengemann) [inline mne/event.py]: just don't do it. no need.

File: mne/event.py (hunk 4 above)
Comment (dengemann) [inline mne/event.py]: yes, this is a good idea.

**Comment by jona-sassenhagen** [comment]
@claire-braboszcz you still working on this? :)

**Comment by claire-braboszcz** [comment]
yes, sorry for the long lag, I had to quit coding in python for a little
while but I should be able to get back to it this month !

**Comment by jona-sassenhagen** [comment]
Ping @claire-braboszcz :)

**Comment by claire-braboszcz** [comment]
Review comments addressed. I could not run the tests locally as `nosetests` segfaults midway (...!). Let see what Travis says.

I still need to add new tests for the feature, though.

**Comment by agramfort** [comment]
@claire-braboszcz travis is not happy

**Comment by agramfort** [comment]
@claire-braboszcz you did not add any test.

shall I close this?

**Comment by claire-braboszcz** [comment]
yes go ahead, sorry for not having finished this !

**Pull request closed without merging**